packages/@tinacms/toolkit/src/components/media/media-manager.tsx

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 import React from 'react'
 import { useEffect, useState } from 'react'
 import styled, { css } from 'styled-components'
-import { useCMS } from '../../react-tinacms'
+import { useCMS } from '../../react-tinacms/use-cms'
```

Comment:
Can you help me understand the difference between importing `useCMS` from `../../react-tinacms/use-cms` versus other places where we import from `tinacms`?

(Example being packages/react-tinacms-github/src/github-client/GithubFile.ts)

Reply:
this is to fix a circular dependency internally within this package. external packages will just be importing from `tinacms`.

the existence of both is likely vestigial at this point now that everything is merged, but that will need to wait for a future refactor.

Reply:
Ah, that makes sense. 👍